dashboard/client/components/+network-services/service-ports.tsx

```diff
@@ -1,4 +1,4 @@
-import "./service-ports.scss"
+import "../pod-ports.scss"
```

Comment:
This should not be imported twice in the modules.
To be consistent please keep imports of style only relevant to the module. 
Otherwise use extra `@import` in SCSS files if you need to re-use some logic/selectors.